src/js/menu/media-chrome-menu.ts

```diff
@@ -141,27 +141,30 @@ template.innerHTML = /*html*/ `
       transform: translate(-100%, 0);
     }
 
-    slot[name="header"] {
-      display: flex;
-      padding: .4em .7em;
-      border-bottom: 1px solid rgb(255 255 255 / .25);
-      cursor: default;
-    }
-
-    slot[name="header"][hidden] {
-      display: none;
-    }
-
-    button[part~="back"] {
+    button {
       background: none;
       color: inherit;
       border: none;
       padding: 0;
       font: inherit;
-      cursor: pointer;
       outline: inherit;
       display: inline-flex;
       align-items: center;
+    }
+
+    slot[name="header"][hidden] {
+      display: none;
+    }
+
+    slot[name="header"] > *,
+    slot[name="header"]::slotted(*) {
```

Comment:
by setting it on the default or slotted elements the styles can be easily overridden